src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -79,20 +79,26 @@ public async Task PostAsync_CancelDuringRequestContentSend_TaskCanceledQuickly(b
         [MemberData(nameof(TwoBoolsAndCancellationMode))]
         public async Task GetAsync_CancelDuringResponseHeadersReceived_TaskCanceledQuickly(bool chunkedTransfer, bool connectionClose, CancellationMode mode)
         {
+            if (LoopbackServerFactory.IsHttp2 && (chunkedTransfer || connectionClose))
+            {
+                // There is no chunked encoding in HTTP/2
```

Comment:
The comment only mentions to skip the test if it is chunked encoding.  But the 'if' statement checks for 'connectionClose'.  How is that related to chunked encoding?  Maybe the comment should be revised?